src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,27 @@
+const Navbar = () => {
+    return (
+        <nav className={'navbar bg-light'}>
```

Comment:
Again, get rid of the curly brackets around all these classNames and just make them strings